client/packages/requisitions/src/RnRForms/DetailView/ContentArea.tsx

```diff
@@ -112,83 +134,17 @@ export const ContentArea = ({
           },
         }}
       >
-        <thead>
-          <tr>
-            <HeaderCell
-              className="sticky-column first-column"
-              label="label.code"
-            />
-            <HeaderCell
-              className="sticky-column second-column"
-              label="label.name"
-            />
-            <HeaderCell label="label.strength" width={85} />
-            <HeaderCell label="label.unit" width={80} />
-            <HeaderCell label="label.ven" width={55} />
-            <HeaderCell
-              label="label.rnr-initial-balance"
-              tooltip="description.rnr-initial-balance"
-            />
-            <HeaderCell
-              label="label.rnr-received"
-              tooltip="description.rnr-received"
-            />
-            <HeaderCell
-              label="label.rnr-consumed"
-              tooltip="description.rnr-consumed"
-            />
-            <HeaderCell
-              label="label.adjusted"
-              tooltip="description.rnr-consumed-adjusted"
-            />
-            <HeaderCell
-              label="label.losses"
-              tooltip="description.rnr-adjustments"
-            />
-            <HeaderCell
-              label="label.rnr-adjustments"
-              tooltip="description.rnr-adjustments"
-            />
-            <HeaderCell label="label.rnr-stock-out-duration" />
-            <HeaderCell
-              label="label.rnr-final-balance"
-              tooltip="description.rnr-final-balance"
-            />
-            <HeaderCell label="label.amc" tooltip="description.rnr-amc" />
-            <HeaderCell
-              label="label.rnr-minimum-quantity"
-              tooltip="description.rnr-minimum-quantity"
-            />
-            <HeaderCell
-              label="label.rnr-maximum-quantity"
-              tooltip="description.rnr-maximum-quantity"
-            />
-            <HeaderCell label="label.expiry" tooltip="description.expiry" />
-            <HeaderCell
-              label="label.requested"
-              tooltip="description.rnr-requested-quantity"
-            />
-            <HeaderCell
-              label="label.low-stock"
-              tooltip="description.rnr-low-stock"
-            />
-            <HeaderCell label="label.comment" />
-            <HeaderCell label="label.confirmed" />
-            <HeaderCell
-              label="label.approved-quantity"
-              tooltip="description.rnr-approved-quantity"
-            />
-          </tr>
-        </thead>
+        <Headers onSearch={onSearch} />
```

Comment:
Moved these to own component, so search stuff changing doesn't cause recalculation of the lines!